ipaserver/install/httpinstance.py

```diff
@@ -57,52 +53,6 @@
 HTTPD_USER = constants.HTTPD_USER
 KDCPROXY_USER = constants.KDCPROXY_USER
 
-# See contrib/nsscipersuite/nssciphersuite.py
-NSS_CIPHER_SUITE = [
```

Comment:
Should we port this into the format of the `SSLCipherSuite` directive?

Reply:
Either we should no longer set a suite at all and leave it to the system policy. Or we should make it a platform decision and don't touch system policies for Fedora and RHEL.

Reply:
I guess it's fine to go with the system policies and possibly add cipher suites in the future in case the defaults are no longer good enough.

Reply:
Right, my goal was to use the system policy.